packages/esm-patient-queues-app/src/home.component.tsx

```diff
@@ -7,18 +7,21 @@ import {
   PRIVILEGE_RECEPTION_QUEUE_LIST,
   PRIVILEGE_TRIAGE_QUEUE_LIST,
 } from './constants';
-import PatientQueueHeader from './patient-queue-header/patient-queue-header.component';
 import ClinicMetrics from './patient-queue-metrics/clinic-metrics.component';
 import QueueLauncher from './queue-launcher/queue-launcher.component';
+import { useTranslation } from 'react-i18next';
+import { HomeHeader } from '@ugandaemr/esm-ugandaemr-commons-lib/src/index';
```

Comment:
i think there is a better way to import . try this `@ugandaemr/esm-ugandaemr-commons-lib`